### PR TITLE
Edited PSR-13 files for small punctuation fixes.

### PR DESCRIPTION
**Pull request by DocFX**
Only non-structural changes, just a few punctuation/grammar fixes.

### CODE REVIEW

File: accepted/PSR-13-links.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Hypermedia links are becoming an increasingly important part of the web, in both
 and various API format contexts. However, there is no single common hypermedia format, nor
 is there a common way to represent links between formats.
 
-This specification aims to provide PHP developers with a simple, common way of representing a
+This specification aims at providing PHP developers with a simple, common way of representing a
```
Comment (Crell) [inline accepted/PSR-13-links.md]: This change is incorrect.  "aims to provide" is valid English grammar.  "aims at providing" I've never seen or heard.

File: accepted/PSR-13-links.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ are not well-standardized or universal, this specification does not seek to stan
 
 For the purposes of this specification, the following definitions apply.
 
-*    **Implementing Object** -An object that implements one of the interfaces defined by this
+*    **Implementing Object** - An object that implements one of the interfaces defined by this
```
Comment (Crell) [inline accepted/PSR-13-links.md]: Legit.

File: accepted/PSR-13-links.md [hunk 3]
```diff
@@ -105,21 +105,21 @@ derived at runtime from some other data source. For that reason, modifiable prov
 are a secondary interface that may optionally be implemented.
 
 Additionally, some Link Provider objects, such as PSR-7 Response objects, are
-by design immutable.  That means methods to add links to them in-place would be
+by design immutable. That means methods to add links to them in-place would be
 incompatible. Therefore, the `EvolvableLinkProviderInterface`'s single method
 requires that a new object be returned, identical to the original but with
-an additional link object included.
+an additional Link object included.
```
Comment (Crell) [inline accepted/PSR-13-links.md]: Legit.

File: accepted/PSR-13-links.md [hunk 4]
```diff
@@ -105,21 +105,21 @@ derived at runtime from some other data source. For that reason, modifiable prov
...
 Link objects are in most cases value objects. As such, allowing them to evolve
 in the same fashion as PSR-7 value objects is a useful option. For that reason,
 an additional EvolvableLinkInterface is included that provides methods to
-produce new object instances with a single change.  The same model is used by PSR-7
+produce new object instances with a single change. The same model is used by PSR-7
 and, thanks to PHP's copy-on-write behavior, is still CPU and memory efficient.
 
-There is no evolvable method for templated, however, as the templated value of a
-link is based exclusively on the href value.  It MUST NOT be set independently, but
+There is no evolvable method for templated values, however, as the templated value of a
```
Comment (Crell) [inline accepted/PSR-13-links.md]: Legit.

**Comment by Crell** [comment]
FIG has no standard on single vs double spaces after a period.  Both are very common in English and both have strong adherents with data on their side.  (I'd argue the double-space has more data on its side, while single has more anecdata from pedants.)  Please leave those alone. :smile:

Other changes are as commented above.

File: accepted/PSR-13-links.md (hunk 1 above)
Comment (DocFX) [reply accepted/PSR-13-links.md]: Well both coexist according to what I find online. I've always seen this version more than the other one. I guess I'll remove that part! :)

Comment (robbieaverill) [reply accepted/PSR-13-links.md]: I'd agree, the original reads correctly

**Comment by localheinz** [comment]
@Crell 

> FIG has no standard on single vs double spaces after a period. Both are very common in English and both have strong adherents with data on their side. (I'd argue the double-space has more data on its side, while single has more anecdata from pedants.) Please leave those alone. 😄

According to https://en.wikipedia.org/wiki/Sentence_spacing, using two spaces after a full stop is clearly backwards-oriented. Apart from not having any effect on the rendering on GitHub or on https://www.php-fig.org/, typing two spaces where one suffices doesn’t make any sense.

**Pull request merged**